compose/project.py

```diff
@@ -67,7 +67,7 @@ def from_config(cls, name, config, client):
         dicts = []
         for service_name, service in list(config.items()):
             if not isinstance(service, dict):
-                raise ConfigurationError('Service "%s" doesn\'t have any configuration options. All top level keys in your fig.yml must map to a dictionary of configuration options.' % service_name)
+                raise ConfigurationError('Service "%s" doesn\'t have any configuration options. All top level keys in your compose.yml must map to a dictionary of configuration options.' % service_name)
```

Comment:
This mentions `compose.yml` , but the name is actually `docker-compose.yml`, right ?